app/Http/routes.php

```diff
@@ -52,7 +52,9 @@
     // Necessary because IDK and please let's fix this @todo
     Route::post('talks/{id}', 'TalksController@update');
     Route::post('conferences/{id}', 'ConferencesController@update');
+    Route::post('bios/{id}', 'BiosController@update');
```

Comment:
The `update` action should be a `PUT` and should be handled by the resource declaration no?